.eslintignore

```diff
@@ -5,4 +5,5 @@ node_modules/*
 packages/application-shell/test-utils/index.js
 packages/application-shell-connectors/test-utils/index.js
 packages/sdk/test-utils/index.js
+packages/application-shell/test-utils/index.js
```

Comment:
?

Reply:
Could be removed. This is built source Similar to the SDK test-utils.

Reply:
I mean it's duplicated (see 3 lines above)